agent/__init__.py

```diff
@@ -209,28 +209,102 @@ def get_ca_cert(debug=False):
     return ca.json()['ca_bundle']
 
 
+def get_mtls_header(debug=False, dev=False):
+    return {
+        'SSL-CLIENT-SUBJECT-DN': 'CN=' + get_device_id(debug=debug, dev=dev),
+        'SSL-CLIENT-VERIFY': 'SUCCESS'
+    } if dev else {}
+
+
+def mtls_req_error_log(request_url, req_type, requester_name, response):
+    """
+    logs error of mtls_request functions
+    :param request_url: request url string
+    :param req_type: 'GET', 'POST', ...
+    :param requester_name: requester id for message, if None then request_url string used
+    :param response - request response
+    :return: None
+    """
+    if not requester_name:
+        requester_name = "({})".format(request_url)
+    print("wott-agent :: mtls_post_request :: [RECEIVED] {} {}: {}".format(
+        requester_name, req_type, response.status_code))
+    print("wott-agent :: mtls_post_request :: [RECEIVED] {} {}: {}".format(
```

Comment:
This will always log `wott-agent :: mtls_post_request ...`, even for `mtls_get_request`.

Reply:
fixed